Newsweek feature

henatan99 opened the pull request:
Pull request for Newsweek page bootstrap clone project, Micorverse. 

In this project,  we were tasked to create a responsive clone for the Newsweek  Magazine homepage, using specifically Bootstrap CSS framework with some Bootstrap Techniques like bootstrap flexbox, bootstrap grid, system, etc.
The project has the following features:

**Mobile View** :
- Navigation bar with a logo, subscribe button, a user log in, and menu tab.
- Main section with bootstrap flex and grid items
- footer with footer nav list items and site logo plus copyright info.

**Tablet View** :
- Navigation bar with a logo, subscribe button, a user log in, and menu tab.
- Main section with bootstrap flex and grid items
- footer with footer nav list items and site logo plus copyright info.


**Laptop  View** :
- Navigation bar with a date, logo, subscribe button, a login, at the top, two nav-list items on above the other.
- Main section with bootstrap flex and grid items
- footer with footer nav list items and site logo plus copyright info

henatan99 committed:
Create repository, add html, css and other files

- Add navbar section and elements
- Add cdn link for bootstrap
- Add scripts for bootstrap and jquery
- Add style using bootstrap

ArreyTabe committed:
styles mob-view header nav-1

henatan99 committed:
Adds main section, section wrapper-1 and child elements

- Adds Section-wrapper-1 element in main section

- Layout section-wrapper-1 child elements using container using bootstrap grid
- Add responsiveness to tablet and laptop screen sizes using bootstrap classes

ArreyTabe committed:
styles magazine section

henatan99 committed:
Adds more bootstrap styles for section-wrapper-1 and other section

- Adds bootstrap styling for section-wrapper-1 elements
- Adds bootstrap classes for ordering elements
- Adds editor's pick section and style using bootstrap
- Adds section-with-aside section and layout it using bootstrap
- Use bootstrap reposnsive classes

ArreyTabe committed:
styles footer

subscribe-newsletter-section

henatan99 committed:
Creates worklows folder and linters.yml file

-Does html and css documnt formtating

henatan99 committed:
Fix webhint and stylelint errors

henatan99 committed:
Fix more stylelint errors in css

henatan99 committed:
FIx more stylelint errors

henatan99 committed:
Adds section-with-aside-elements section, do more styling

- Adds more bootstrap styling for the subscribe, newsletter and footer sections
- Adds responsive features for the sections

henatan99 committed:
Fix minor stylelint errors

henatan99 committed:
Removes eol empty line more than one

henatan99 committed:
Removes eol more than one empty line

henatan99 committed:
Fix unexpected errors

henatan99 committed:
Adds stylelint, hintrc and gitignore files

- Make minor changes on readme files

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                <span class="footer-social-icons d-flex">
+                    <a href="#">
+                        <span class="iconify" data-icon="ant-design:facebook-filled" data-inline="false"></span>
+                    </a>
+                    <a href="">
+                        <span class="iconify" data-icon="ant-design:twitter-outlined" data-inline="false"></span>
+                    </a>
+                    <a href="#">
+                        <span class="iconify" data-icon="raphael:instagram" data-inline="false"></span>
+                    </a>
+                    <a href="#">
+                        <span class="iconify" data-icon="cib:tumblr" data-inline="false"></span>
+                    </a>
+                    <a href="#">
+                        <span class="iconify" data-icon="bx:bxl-linkedin" data-inline="false"></span>
+                    </a>
+                </span>
+            </div>
+            <hr>
+            <div class="footer-links d-flex flex-wrap align-items-baseline mb-4">
+                Editions:
+                <a href="#"> U.S. Edition</a>
+                <a href="#">日本</a>
+                <a href="#">Pakistan</a>
+                <a href="#">Polska</a>
+                <a href="#">România</a>
+                <a href="#"> About Us</a>
+                <a href="#">Corrections</a>
+                <a href="#">Contact Us</a>
+                <a href="#">Editorial Guidelines</a>
+                <a href="#"> Advertise</a>
+                <a href="#">Copyright</a>
+                <a href="#">Terms & Conditions</a>
+                <a href="#">Privacy Policy</a>
+                <a href="#"> Cookie Policy</a>
+                <a href="#">Terms of Sale</a>
+                <a href="#">Archive</a>
+                <a href="#">Announcements</a>
+            </div>
+        </footer>
```
- [x] Please move footer section outside main section

henatan99 replied:
footer section out of main section

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                <article>
+                    <figure class="position-relative">
+                        <a href="#">
+                            <img src="assets/images/top-story/nancy-pelosi.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/trump.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="">
+                                <img src="assets/images/featured-section/greece-turkey-military-dispute-mediterranean.jpg"
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/judge-amy-coney-barrett.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/russia-ryabkov-nuclear-weapons-talks.jpg"
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        </h2>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/bruce-abramson-jeff-ballabon.png" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/david-p-goldman.png" alt="" style="padding: 0;">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/harry-j-kazianis.png" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/jennifer-weiss-wolf.png" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/mike-davis.png" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/richards-axe-briggs.png" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/robert-reich.png" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        <hr>
+                        <article class="d-flex">
+                            <a href="#">
+                                <img src="assets/images/opinion/shoshana-bryen.png" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/holy-angel-university.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/americas-best-banks-2021.png" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/worlds-best-specialized-hospitals-2021.png"
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/best-online-shops-2021.png" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/americas-best-physical-rehabilitation-centers-2020.png"
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/featured-section/brandspark.png" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <div class="row">
+                        <article class="col-12 col-md-4 mr-n2 ">
+                            <a href="#">
+                                <img src="assets/images/my-turn/doctors-black-doctors-black-women.jpg" alt=""
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        </article>
+                        <article class="col-12 col-md-4 ml-n1 mr-n1">
+                            <a href="#">
+                                <img src="assets/images/my-turn/chef-cooking-tiktok-pandemic-coronavirus.png" alt=""
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        </article>
+                        <article class="col-12 col-md-4 ml-n2">
+                            <a href="#">
+                                <img src="assets/images/my-turn/love-romance-italy-romeo-juliet-pandemic.jpg" alt=""
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        <article class="col-12 col-md-4 mr-n2 ">
+                            <figure>
+                                <a href="#">
+                                    <img src="assets/images/culture-and-travel/singapore-airlines.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        <article class="col-12 col-md-4 ml-n1 mr-n1">
+                            <figure>
+                                <a href="#">
+                                    <img src="assets/images/culture-and-travel/100-best-rock-albums-all-time-according-critics.png"
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        <article class="col-12 col-md-4 ml-n2">
+                            <figure>
+                                <a href="#">
+                                    <img src="assets/images/culture-and-travel/cul-map-wes-anderson-white-cyclon-banner.jpg"
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/downtown-broadway-tennessee-coronavirus-august-2020.jpg"
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/houston-ballot-drop-off.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/mages/more-stories/lindsey-graham.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/protest-chicago.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/elderly-mask.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/masks-coronavirus-covid-getty-stock.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/china-xi-jinping.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/protest-chicago.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/meghan-markle-admiralty-house-sydney-australia.jpg"
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/vladimir-putin.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/former-secretary-state-hillary-clinton.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <hr>
+                    <article>
+                        <a href="#" class="float-left pr-3">
+                            <img src="assets/images/more-stories/us-president-donald-trump.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/rankings-and-sponored/americas-best-addiction-treatment-centers-2020.png"
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/rankings-and-sponored/americas-best-nursing-homes-2020.png"
+                                    alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/rankings-and-sponored/gravely-pc.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/rankings-and-sponored/image-3-this-startup-making.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat commented on index.html:
```diff
@@ -0,0 +1,1491 @@
+                    <article>
+                        <figure>
+                            <a href="#">
+                                <img src="assets/images/rankings-and-sponored/savannah-college-art-design.jpg" alt="">
```
- [ ] Please use text for the **alt** attribute.

arslanbisharat reviewed (changes requested):
### Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

                                      ### Project Specific 

- [x]  Please make your PR descriptive. Please add more details to it. Please refer to [link ](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642) from the pathway to learn more. 

- [x] Your CSS has too many font-size declarations (12), abstraction needed.

- [ ] ![bootstrap_classes_requirement](https://user-images.githubusercontent.com/58504655/96834751-fd594e80-145b-11eb-8587-3796b46c0732.png)

- [x]  Please use dry CSS wherever needed. I have given some hints through inline-comments below. Please work on it. 

- [ ]  In Bootstrap projects make sure to less use the styles.css file for styles that could be done using Bootstrap classes since that's the main goal of the exercise. Things like font-weight, flex justifications, display, etc could be implemented completely with Bootstrap classes. [Please refer to the main page of bootstrap to learn more about bootstrap. ](https://getbootstrap.com/docs/4.5/getting-started/introduction/). I have given some hints through inline-comments below. Please work on it. 

                                               ### Large Screen

- [x] In the navbar doesn't look like the original one. Please have a look below for suggestions. 

**Clone**

![chrome-capture (7)](https://user-images.githubusercontent.com/58504655/96835535-24fce680-145d-11eb-959d-07cc2b04f81b.jpg)

- [x] There should no white space in this section.

**clone**

![chrome-capture (8)](https://user-images.githubusercontent.com/58504655/96835714-642b3780-145d-11eb-8662-fab84a5e6966.jpg)

- [x] This section doesn't look like the original one. 

**clone**

![chrome-capture (9)](https://user-images.githubusercontent.com/58504655/96835852-9177e580-145d-11eb-9e9a-fabbe431d416.jpg)

- [x] There should less space. 

**clone**

![chrome-capture (10)](https://user-images.githubusercontent.com/58504655/96836086-e7e52400-145d-11eb-984f-8bd2017ad4da.jpg)

- [x] There should total of 8 items in **EDITOR'S PICK** but you have only 7. The heading of EDITOR'S PICK doesn't look like the original one. 

- [x] You have extra space in all of these sections please remove that. 

![chrome-capture (11)](https://user-images.githubusercontent.com/58504655/96836391-588c4080-145e-11eb-990c-493b15942739.jpg)

- [ ] Subscribe section should have more width than the newsletter section. n newsletter there should space between email input and signup button. 
![chrome-capture (12)](https://user-images.githubusercontent.com/58504655/96836526-896c7580-145e-11eb-974c-73405de0158b.jpg)

- [ ] ![chrome-capture (13)](https://user-images.githubusercontent.com/58504655/96836763-dcdec380-145e-11eb-8a3c-a6b86e49cd75.jpg)

- [ ] Please male Edition bold, and add proper spacing and | after each item. 

**clone**

![chrome-capture (14)](https://user-images.githubusercontent.com/58504655/96836874-05ff5400-145f-11eb-8d1d-d78ee48c8ed2.jpg)


                                             ### MEDIUM SCREEN

- [ ] Please male Edition bold, and add proper spacing and | after each item. 

**clone**

![chrome-capture (14)](https://user-images.githubusercontent.com/58504655/96836874-05ff5400-145f-11eb-8d1d-d78ee48c8ed2.jpg)

                                         ###SMALL SCREEN

- [x] On the small screen, starting from the center of the page, the content is neither properly aligned nor has proper spacing. There is an overlapping of text. Please fix that. 



**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

henatan99 committed:
Fix some errors from TSE's code review

- Fix overlapping sections and alignment issues in mobile screen
- Re-locate the footer section out of the main section

ArreyTabe committed:
fixes PR errors

edits the fully descriptive PR

removes white spaces from sections

add strokes to footer list items

ArreyTabe committed:
updates readme

ArreyTabe committed:
fixes broken image

ArreyTabe committed:
Merge branch 'newsweek-feature' of github.com:henatan99/NewsWeek-page-bootstrap-clone into newsweek-feature

henatan99 committed:
Fix some styling errors

ArreyTabe committed:
fixes page layout errors

ArreyTabe committed:
fixes linters errors

ArreyTabe committed:
linters errors fix

arslanbisharat commented on README.md:
```diff
@@ -1,2 +1,71 @@
-This project is about building a replica of NewsWeek page using bootstrap framework and making it responsive. 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HTML form page 
```
remove this

arslanbisharat reviewed (changes requested):
### Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [ ] On the large screen, the webpage has white space around the borders. Please work on it 

- [ ]  In the navbar doesn't look like the original one. Please have a look below for suggestions.
![chrome-capture (25)](https://user-images.githubusercontent.com/58504655/96963693-833fcd00-1522-11eb-90b1-788f56e75731.jpg)

- [ ] Please remove white space from here by adding more items
![img](https://user-images.githubusercontent.com/58504655/96835714-642b3780-145d-11eb-8662-fab84a5e6966.jpg)

- [ ] Newsletter has more image height than the subscribe section. 

![chrome-capture (26)](https://user-images.githubusercontent.com/58504655/96963918-f47f8000-1522-11eb-859c-e4519ecf6f5e.jpg)

- [ ] On tablet mode text of the items present in the center is present vertically. Please work on it. 

![chrome-capture (28)](https://user-images.githubusercontent.com/58504655/96964138-5cce6180-1523-11eb-98e6-195370938806.jpg)


- [ ] On small screen image should be in the center 
![chrome-capture (27)](https://user-images.githubusercontent.com/58504655/96964022-242e8800-1523-11eb-9e30-60918254ae7e.jpg)


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

arslanbisharat reviewed (changes requested):
# Requested Changes

- [ ] Please make changes asked in previous reviews. You have not made a single commit after your second review. 
In case of any question please reach me out on slack @arslanbisharat

henatan99 committed:
Changes made based on TSE's review

henatan99 committed:
Resolve file conflict

henatan99 committed:
Resolves file conflict

henatan99 committed:
Merge branch 'newsweek-feature' of https://github.com/henatan99/NewsWeek-page-bootstrap-clone into newsweek-feature

henatan99 committed:
Fix file comflict

henatan99 committed:
Fix lintstyle errors

henatan99 committed:
Fix horizontal sliding bar issue

- Horizontal bar slides beacuse of the subscribe infor has improper
  padding misbehaving in media queries

henatan99 committed:
Fix empty spaces in columns and spacing in subscribe section

henatan99 committed:
Fix stylelint errors

arslanbisharat reviewed (approved):
#  Approved

Thank you for making those changes requested by previous TSEs.
You can merge and move forward with your curriculum.

Regards,
Arslanbisharat
